src/app/app.component.ts

```diff
@@ -66,10 +66,12 @@ export class AppComponent implements OnInit, AfterViewInit {
 
   // Helper methods for fetching AppView state
   showReconnectView() {
+    return false;
     return this.appView === AppView.RECONNECT;
   }
 
   showAppView() {
```

Comment:
needs a type

Reply:
Should be inferred by the compiler; want it typed explicitly in the function signature?

Reply:
Yeah, let's have it typed